### DISCUSSION TITLE
Like to contribute - what I can start with?

### DISCUSSION THREAD

**Discussion by Divine369** [flet-dev/flet#2804] [General]
Hello,

I am a python programmer coming from embedded systems background with knowledge of C, C++, JavaScript and Python. (8 years)
Also very experienced with PySide6 widgets and core components. Very basic knowledge of Dart and Flutter (upto check list app).
I was looking for making a mobile application using python and came across Flet just today. So very new to this community.
But I will love to contribute as I am very sure Flet is going to be very helpful to the community.
People are looking for exactly this. But I can see there is lot to do still.
If there is anyone who can direct me - I am open for contribution.

Divine

**Comment by FeodorFitsner** [comment]
Great, thanks for offering your help - much appreciated! :)

Right now there are two areas where C/C++ skill would be helpful:
1) Improving communication between embedded Python and Flutter. Right now sockets are used, but if there is a chance Python and Flutter could talk directly (native callbacks) we'd like to use that. It requires good understanding of threading model, FFI, maybe making a native module for Python.
2) Making/updating "recipes" for building Python binary wheels for iOS and Android. There is compiler/linker/patching wizardry to build C sources against iOS/Android.

I can give more details on Discord - DM me there or we could continue here, create a new issue.
